src/main/resources/org/jenkinsci/plugins/envinject/EnvInjectBuildWrapper/help-propertiesContent_ja.html

```diff
@@ -0,0 +1,7 @@
+<div>
+    <p>
+        KEY=VALUE形式で、キーと値を設定します。<br />
+        環境変数にプロパティの名前でアクセスできます。<br />
+        上記のプロパティファイルで設定したプロパティを使用したり、上書きすることも可能です。
```

Comment:
I suspect it's the original message. It makes sense to remove it (users will get the English text) or ask somebiody like @kohsuke to translate it. Otherwise users with Japanese locale will get a message without sensitive details